backend/EkoFunkcje/EkoFunkcje/Features/Interventions/GetAllGeoInterventionsFunction.cs

```diff
@@ -1,23 +1,22 @@
-using AutoMapper;
-using Microsoft.AspNetCore.Http;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using EkoFunkcje.Models;
+using EkoFunkcje.Models.Respones;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.Extensions.Logging;
-using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Table;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
-namespace EkoFunkcje
+namespace EkoFunkcje.Features.Interventions
 {
-    public static class GetAllGeoDataFunction
+    public static class GetAllGeoInterventionsFunction
     {
-        [FunctionName("GetAllGeoDataFunction")]
+        [FunctionName("GetAllGeoInterventions")]
         public static async Task<IActionResult> Run(
-            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req, 
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "GetAllGeoInterventions")] HttpRequestMessage req, 
```

Comment:
Tutaj nie bardzo wiem co robi ta funkcja, ale wrócę do niej w wolnej chwili.

Reply:
Ok, z tego co rozumiem, to ta funkcja miała dostarczać dane geograficzne punktów na mapie, ale chyba na froncie z niej nie skorzystaliśmy w czasie hackathonu. Zastanawiam się, czy jest nam w ogóle potrzebna. Aktualnie dane geograficzne są zapisywane w funkcji dla tworzenia interwencji.
Z drugiej strony - jakbyśmy chcieli uzyskać interwencje dla danego prostokąta mapy, to moglibyśmy zrobić zapytanie do endpointu od wszystkich interwencji z query parametrami typu `interventions?latMax=val1&latMin=val2&llngMax=val3=lngMin=val4` i otrzymać interwencje zawierające się wewnątrz prostokąta opisanego tymi współrzędnymi. Co o tym myślisz?

I kolejna sprawa, chociaż może to w kolejnych ewentualnych wersjach apki - może walidowanie adresów i pobieranie danych na temat współrzędnych geograficznych powinnismy przenieść na front? Wtedy moglibysmy skorzystać np. z geolokalizacji urządzenia jeśli użytkownik wyraził by na to zgodę oraz dać użytkownikowi natychmiastową informację zwrotną jeśli się pomylił przy adresie itp.

Reply:
Darek będzie zmieniał metody pobierania interwencji na mape